cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/StateVariablesExport.java

```diff
@@ -316,10 +299,6 @@ private static void writePowerFlowTerminalFromProperty(Identifiable<?> c, String
     }
 
     private static void writePowerFlow(String cgmesTerminalId, double p, double q, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) {
```

Comment:
If we remove the condition for missing flows here we are writing as p=q=0 many power flows for terminals of lines and transformers that have not been initialized. I think we should avoid writing an SvPowerFlow object if both values are undefined. Writing power flow values for terminals of branches is not required.

Reply:
SvPowerFlow are required in Quality requirements (at least, their absences are considered as warnings)

```xml
<qodc:RuleModel qodc:ruleID="SvPowerFlowInstances" qodc:severity="ERROR">
		<qodc:Level>5</qodc:Level>
		<qodc:RuleDetails>
		SvPowerFlow class is required to be instantiated for the following classes:
		subclasses of the RotatingMachine, subclasses of the EnergyConsumer, 
		EquivalentInjection, ShuntCompensator, StaticVarCompensator and EnergySource.
		</qodc:RuleDetails>
```

```xml
<qodc:RuleModel qodc:ruleID="SvPowerFlowBranchInstances" qodc:severity="WARNING">
		<qodc:Level>5</qodc:Level>
		<qodc:RuleDetails>
		SvPowerFlow class is required to be instantiated for branches if 
		referenced by a TieFlow.
		</qodc:RuleDetails>
```

Reply:
Yes, they are required in general for injections (both P and Q may be results of power flow calculations) and tie flows. but the quality requirements do not consider power flows to be present in SV for lines and transformers that are not tie flows. 

My problem here is that if we have NaNs in IIDM, it means we have not the set or calculated the value in IIDM. We will write the SV a value that may not be correct.

Reply:
> quality requirements do not consider power flows to be present in SV for lines and transformers that are not tie flows

Are you sure? This is not what I gathered from the rule I quoted above...

> My problem here is that if we have NaNs in IIDM, it means we have not the set or calculated the value in IIDM. We will write the SV a value that may not be correct.

I agree but I'm not seeing another default value which would "more correct"

Reply:
You're right! So we ignore lines and transfo that do not reference tie flows